src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier supplier = new ColorSupplier();
+        Random random = new Random();

```

Comment:
lets make Random and ColorSupplier a class-level fields - just like Random in ColorSupplier